### PR TITLE
[WIP] Create new frontpage for logged in users

### PR DESCRIPTION
**Pull request by PeterJFB**
### Loading new frontpage ...

Preview-images are below, all feedback is appreciated :100: 

### PRs
However, commenting on code will be done in individual PRs and merged into the `new-frontpage-2022-release`-branch, please have a look:
- webkom/lego#2594
- #2696 
- more coming ...

## Current look (29.04):

image
image

### 50% (29.04)

<details>
  <summary>Click to expand!</summary>

image
image
image
</details>

### Mobile (29.04)
<details>
  <summary>Click to expand!</summary>

image
image
image
</details>

### Dark (29.04)

image
image

### Poll (<29.04)
Peek 2022-01-25 17-14

### CODE REVIEW

File: app/components/Poll/Poll.css [hunk 1]
```diff
@@ -1,172 +1,172 @@
...
-.pollGraph span {
-  vertical-align: middle;
+.voteButton {
+  background: var(--lego-red);
+  color: var(--color-white);
+  font-weight: 500;
+  height: 37px;
+  width: 90%;
+  padding: 5px;
+  margin: 2px !important;
```
Comment (LudvigHz) [inline app/components/Poll/Poll.css:73]: :exclamation: :no_good:

File: app/components/RandomQuote/RandomQuote.css [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
+  color: var(--lego-link-color);
+}
+
 .quoteReactions {
   margin-top: 5px;
 }
 
 .actions {
-  margin: 5px 5px 5px 15px;
+  margin: 5px 5px 5px 5px;
```
Comment (LudvigHz) [inline app/components/RandomQuote/RandomQuote.css:23]: === `margin: 5px;`

File: app/components/RandomQuote/RandomQuote.css [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
 }
 
-.fetchNew {
+.actions * {
+  line-height: 0;
+  margin: 0;
   margin-bottom: 5px;
 }
+
+.actions * + * {
```
Comment (LudvigHz) [inline app/components/RandomQuote/RandomQuote.css:32]: uhhhh, any way to make this a bit easier to understand?

File: app/routes/overview/components/BlankCard.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import cx from 'classnames';
+import styles from './BlankCard.css';
+
+import Flex from '../../../components/Layout/Flex';
```
Comment (LudvigHz) [inline app/routes/overview/components/BlankCard.js:4]: use absolute paths

File: app/routes/overview/components/Overview.css [hunk 5]
```diff
@@ -34,14 +34,50 @@
...
+  }
+}
+
+.pollWrapper {
+  position: relative;
+  overflow-y: visible;
+}
+
+.pollWrapper > * {
+  position: absolute !important;
```
Comment (LudvigHz) [inline app/routes/overview/components/Overview.css:74]: :no_good:

File: app/routes/overview/components/Overview.js [hunk 6]
```diff
@@ -16,7 +16,13 @@ import NextEvent from './NextEvent';
 import Poll from 'app/components/Poll';
 import type { PollEntity } from 'app/reducers/polls';
 import RandomQuote from 'app/components/RandomQuote';
+import FancyNodesCanvas from '../../../../app/components/Header/FancyNodesCanvas';
```
Comment (LudvigHz) [inline app/routes/overview/components/Overview.js:19]: `app/components/...`

Comment (PeterJFB) [reply app/routes/overview/components/Overview.js]: Nicely spotted! All suggestions above are now implemented^

**Comment by SmithPeder** [comment]
I feel like these "cards" need the same border-radius
image

as these cards
image

That will make the whole page more consistant

**Comment by PeterJFB** [comment]
> One of these is supposed to be an event right? I feel like this is too much "article".

Correct! I'll update them as soon as i can

> I feel like these "cards" need the same border-radius
> That will make the whole page more consistent

Damn, didn't think of that. It wouldn't hurt with some more consistency 👌

**Comment by PeterJFB** [comment]
Moved permanently to #2697

**Pull request closed without merging**